Gopkg.toml

```diff
@@ -93,6 +93,11 @@
   name = "github.com/golang/protobuf"
   version = "=1.1.0"
 
+# We need to specify fsnotify source to avoid dep panic
```

Comment:
Could you elaborate on why this happens.

Reply:
E.g. https://github.com/kubernetes-sigs/kubebuilder/issues/316#issue-342173819

Reply:
Please could you add your answer to your comment in Gopkg.toml.

Reply:
https://github.com/openshift/cluster-api-provider-libvirt/pull/39/commits/1dd0ba2d7d91ef8a034b7bdd8ab919c26a380cc3#diff-836546cc53507f6b2d581088903b1785R97